app/src/main/res/values-de/strings.xml

```diff
@@ -239,4 +239,5 @@
     <string name="content_description_notification_count">Anzahl Meldungen</string>
     <string name="content_description_members_marker_map">Mitglieder</string>
     <string name="content_description_member_marker_map">Mitglied</string>
+    <string name="label_debug">Warmshowers (Debug)</string>
```

Comment:
Can you please squash the changes of this commit (16208bb) into 0fb2783? Thank you!

Reply:
When using separate `strings.xml` per `build type`, this commit will be gone.